Source/cursor.cpp

```diff
@@ -261,7 +261,7 @@ void CheckCursMove()
 	pcursplr = -1;
 	uitemflag = 0;
 	panelflag = 0;
-	trigflag_3 = 0;
+	trigflag = 0;

```

Comment:
```suggestion
	trigflag = FALSE;
```
there's a few of these